taichi/python/export_lang.cpp

```diff
@@ -285,6 +286,7 @@ void export_lang(py::module &m) {
       .def("insert_external_func_call", &ASTBuilder::insert_external_func_call)
       .def("expr_alloca", &ASTBuilder::expr_alloca)
       .def("expr_alloca_local_tensor", &ASTBuilder::expr_alloca_local_tensor)
+      .def("expr_alloca_matrix", &ASTBuilder::expr_alloca_local_matrix)
```

Comment:
Is `expr_alloca_matrix` doing the same thing as `expr_alloca_local_tensor`? Then can we make `expr_alloca_matrix` an alternative code branch controlled by "real_matrix" inside `expr_alloca_local_tensor`?

This way, we hide implementation details to Python